indico/web/client/js/tinymce.js

```diff
@@ -99,6 +100,55 @@ export const getConfig = (
     {start: '* ', cmd: 'InsertUnorderedList'},
     {start: '- ', cmd: 'InsertUnorderedList'},
   ],
+  formats: {
+    mark: {inline: 'mark'},
+  },
+  // This is the default, with "Mark" added to the list.
+  // https://www.tiny.cloud/docs/tinymce/latest/user-formatting-options/#style_formats
+  style_formats: [
```

Comment:
I don't replicating the default here, but `style_formats_merge` only works for top-level entries, so I can't add something to the "Inline" list that way...

Reply:
Checked the code, and unfortunately the default styles are not exported in any way :/